CardanoSharp.Wallet/Models/Transactions/TransactionBody/TransactionOutputValue.cs

```diff
@@ -4,7 +4,7 @@ namespace CardanoSharp.Wallet.Models.Transactions
 {
     public partial class TransactionOutputValue
     {
-        public ulong Coin { get; set; }
```

Comment:
This is the only one that I'm unsure about. This implies we can send negative ADA

Reply:
Makes sense, I will update this in a few hours